src/main/java/com/reservation/studio/domain/BusinessHoursChanged.java

```diff
@@ -0,0 +1,9 @@
+package com.reservation.studio.domain;
+
+import java.time.Instant;
+
+record BusinessHoursChanged(StudioId studioId, BusinessHours businessHours, Instant changedAt) implements StudioEvent {
```

Comment:
wydaje mi się, że lepiej będzie:
```suggestion
record BusinessHoursChanged(StudioId studioId, BusinessHours businessHours, LocalDateTime changedAt) implements StudioEvent {
```
Miałem tę rozkminę przy refaktorze Instruktora, bo tam w zdarzeniach też było używane `Instant`.

Faktycznie `Instant` lepiej nadaje się do zdarzeń, bo reprezentuje unikalny moment w czasie, razem z informacją o strefie czasowej.
Tylko, że nasze zdarzenia tego nie będą potrzebować, bo domena korzysta cały czas z `LocalDateTime` i zakładamy, że działa w jednej strefie czasowej.
Nawet trochę nam to utrudni sprawę, bo jeśli w jakimś serwisie domenowym będziemy konsumować te zdarzenia, to cały czas będzie trzeba robić konwersję pomiędzy `Instant` i `LocalDateTime`.
Jedyny sens dla `Instant` jaki widzę, to gdyby nasze zdarzenia pochodziły z wielu mikroserwisów, które działają w kilku strefach czasowych i były gdzieś agregowane 🤔

Of course nie upieram się przy swoim, bo może o czymś nie wiem i `Instant` faktycznie będzie lepszy 😁
Ale zdecydujmy się przynajmniej na jedną opcję, bo we wzorcowym agregacie zdarzenia używają `LocalDateTime` 😉